### ISSUE TITLE
Move cryptography to the service running in separate process

### ISSUE THREAD

**Issue by siepra** [ZbayApp/ZbayMobile#88]
Main app along with notifications task should communicate with the service responsible for messages decryption (which will be a separate nodejs process run from native code) with the use of websockets. Thanks to that we'll be able to get rid of react-native-webview-crypto (which is slowing down the app as it runs web browser under the hood). Also this is the best option to make it possible to display notifications about direct messages (as HeadlessJsTask which we use for handling updates from waggle is not capable of using user interface - webview in this particular case).

**Comment by holmesworcester** [comment]
@Dinek007 can you confirm that signature verification is moved to the backend service by #956? If so I think we can close this issue. 

Also is signing still done in a webview? Or is that done in the backend as well?

**Comment by Dinek007** [comment]
> 

yes signature verification is moved to backend but signing is in webview
